handsontable/src/editorManager.js

```diff
@@ -99,6 +88,62 @@ class EditorManager {
     this.instance.view.wt.update('onCellDblClick', (event, coords, elem) => this.onCellDblClick(event, coords, elem));
   }
 
+  registerShortcuts() {
```

Comment:
Can I ask you to add jsdoc tags to this method?

Reply:
Done within 30829ab5f1f4848cd5462c661bf4c11ef87312df.